scripts/update.sh

```diff
@@ -166,3 +169,20 @@ update_moonraker(){
   ok_msg "Update complete!"
   start_moonraker
 }
+
+update_klipperscreen(){
+  stop_klipperscreen
+  cd $KLIPPERSCREEN_DIR
+  KLIPPERSCREEN_OLDREQ_MD5SUM=$(md5sum $KLIPPERSCREEN_DIR/scripts/KlipperScreen-requirements.txt | cut -d " " -f1)
+  git pull origin master -q && ok_msg "Fetch successfull!"
+  git checkout -f origin/master && ok_msg "Checkout successfull"
+  #KLIPPERSCREEN_NEWREQ_MD5SUM=$(md5sum $KLIPPERSCREEN_DIR/scripts/KlipperScreen-requirements.txt)
+  if [[ $(md5sum $KLIPPERSCREEN_DIR/scripts/KlipperScreen-requirements.txt | cut -d " " -f1) != $KLIPPERSCREEN_OLDREQ_MD5SUM ]]; then
+    status_msg "New dependecies detected..."
+    PYTHONDIR="${HOME}/.KlipperScreen-env"
+    $PYTHONDIR/bin/pip install -r $KLIPPERSCREEN_DIR/scripts/KlipperScreen-requirements.txt
+    ok_msg "Dependencies have been installed!"
+  fi
+  ok_msg "Update complete!"
+  start_klipper
```

Comment:
Do we maybe need a `start_klipperscreen `here instead?
I haven't seen a `stop_klipper `call before... but only a `stop_klipperscreen`.